gen/compile_proto.jl

```diff
@@ -61,5 +102,7 @@ append!(files_to_include, (process_module("tensorboard/plugins/$plugin", base_mo
 # files_to_include contains all the proto files, can be used for printing and inspection
 println("generated code for \n$files_to_include")
 
+clean_files(out_dir)
```

Comment:
Let's leave this for a separate PR. This censoring is redundant and messes up file paths.
```suggestion
# clean_files(out_dir)
```

This censoring can be done with a shell script:
`>>> cat file_list | xargs -I f sed -i 's|/home/lior||g' f`

Reply:
Likewise the `_pb.jl` files can be restored to their previous version with the full path